### ISSUE TITLE
Can't run xcat

### ISSUE THREAD

**Issue by gsdt** [orf/xcat#25]
Please help me with this error when start run xcat:
```
Traceback (most recent call last):
  File "/usr/local/bin/xcat", line 9, in <module>
    load_entry_point('xcat==0.9', 'console_scripts', 'xcat')()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 542, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2569, in load_entry_point
    return ep.load()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2229, in load
    return self.resolve()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2235, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python3.5/dist-packages/xcat-0.9-py3.5.egg/xcat/cli.py", line 40, in <module>
  File "/usr/local/lib/python3.5/dist-packages/xcat-0.9-py3.5.egg/xcat/shell.py", line 8, in <module>
ImportError: cannot import name 'prompt_async'

```

**Comment by gabontau** [comment]
Had the same issue, but solved by carefully checking other issues.

**Comment by orf** [comment]
I'm aware of this and I'm working on it. I've redesigned the internals, current master is working-ish but still needs work (OOB and shell).

**Comment by orf** [comment]
Hello,
I've just released `xcat` 1.0 on pypi 🎉 . I've redesigned the CLI and it now all works. Please see the updated README for information.

I'm working on the documentation now. Please open an issue if you run into trouble.
